### ISSUE TITLE
soup: add EOL reminder

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/security-onion#1803]
Remind users of EOL date for Security Onion 16.04.

**Comment by dougburks** [comment]
`soup` now includes the following note:
```
###########################################################################
PLEASE NOTE!
Ubuntu 16.04 reaches EOL in April 2021 and so therefore Security Onion 16.04 does as well.
We will not provide any support for Security Onion 16.04 after April 16, 2021.
Please plan to upgrade or replace any existing 16.04 systems before then.
https://docs.securityonion.net/en/16.04/eol.html
###########################################################################
```

`securityonion-sostat - 20120722-0ubuntu0securityonion147` is now ready for testing at `ppa:securityonion/test`.

**Comment by cm-ops** [comment]
Verified the banner note appears when `soup` is run.

**Comment by dougburks** [comment]
Thanks @cm-ops !

**Comment by dougburks** [comment]
securityonion-sostat - 20120722-0ubuntu0securityonion148 now available for Security Onion 16.04!
https://blog.securityonion.net/2020/12/securityonion-sostat-20120722.html
